### PR TITLE
[Feat][kubectl-plugin] Add kubectl ray version command

### PR DESCRIPTION
**Pull request by MortalHappiness**
## Why are these changes needed?

See the description in the corresponding issue for details.

## Related issue number

Closes: ray-project/kuberay#2423

## Checks

- [x] I've made sure the tests are passing.
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: kubectl-plugin/pkg/cmd/version/version.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+	"github.com/spf13/cobra"
+)
+
+var Version = "development"
+
+func NewVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Show kubectl ray version",
```
Comment (andrewsykim) [inline kubectl-plugin/pkg/cmd/version/version.go:14]: nit: "Output the version of the Ray kubectl plugin"

Comment (MortalHappiness) [reply kubectl-plugin/pkg/cmd/version/version.go]: done

File: kubectl-plugin/.goreleaser.yaml [hunk 2]
```diff
@@ -16,6 +16,7 @@ builds:
       - arm64
     main: ./cmd
     binary: kubectl-ray
+    ldflags: -X github.com/ray-project/kuberay/kubectl-plugin/pkg/cmd/version.Version={{- .Tag -}}
```
Comment (kevin85421) [inline kubectl-plugin/.goreleaser.yaml:19]: what is the value of `.Tag`?

File: kubectl-plugin/pkg/cmd/version/version.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var Version = "development"
```
Comment (kevin85421) [inline kubectl-plugin/pkg/cmd/version/version.go:9]:
```suggestion
const Version string = "development"
```

**Review by kevin85421 — COMMENTED** [review]
can you add a screenshot for the result of this command?

File: kubectl-plugin/pkg/cmd/version/version.go (hunk 3 above)
Comment (MortalHappiness) [reply kubectl-plugin/pkg/cmd/version/version.go:9]: This cannot be replaced with const. This variable will be override by ldflags at build time.

File: kubectl-plugin/.goreleaser.yaml (hunk 2 above)
Comment (MortalHappiness) [reply kubectl-plugin/.goreleaser.yaml:19]: A git tag, semver tag. e.g. `v1.0.0`

**Comment by MortalHappiness** [comment]
image

**Review by kevin85421 — APPROVED** [review]

**Review by andrewsykim — APPROVED** [review]

**Pull request merged**